seat.c

```diff
@@ -407,6 +408,28 @@ handle_new_keyboard(struct cg_seat *seat, struct wlr_keyboard *keyboard, bool vi
 	wlr_seat_set_keyboard(seat->seat, keyboard);
 }
 
+static void
+handle_set_gamma(struct wl_listener *listener, void *data)
```

Comment:
Is it really related to seat? Maybe you could move the code to output.c instead.